src/Temporary/TransClient.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\ObjectEncryptor\Temporary;
+
+use Keboola\AzureKeyVaultClient\Client;
+use Keboola\AzureKeyVaultClient\GuzzleClientFactory;
+
+class TransClient extends Client
+{
+    public function __construct(GuzzleClientFactory $clientFactory, ?string $encryptorId)
+    {
+        $vaultBaseUrl = (string) getenv(self::determinateVaultUrlEnvName($encryptorId));
+
+        if ($vaultBaseUrl === '') {
+            throw new TransClientNotAvailableException;
+        }
+
+        parent::__construct(
+            $clientFactory,
+            new TransAuthenticatorFactory(),
+            $vaultBaseUrl,
+        );
+    }
+
+    public static function determinateVaultUrlEnvName(?string $encryptorId): string
```

Comment:
tomuhle nerozumím. proč je ten název envu dynamický? a nakonec to ani není použitý v AKVWrappersWithTransClientTest

Reply:
proč prostě nestačí udělat getEnv(TRANS_AZURE_KEY_VAULT_URL) ?

Reply:
Kvůli více AKV v `sandboxes-service`: https://github.com/keboola/sandboxes-service/blob/0a8e2d0137554e91efe7e2a0eb358829fb445eb7/config/services.yaml#L180

Reply:
Doděláno, rebasnul jsem to do [posledního commitu](https://github.com/keboola/object-encryptor/pull/51/commits/4dc955a25c86d1f055045b64a5e0281216d81388#diff-5f33c31b6fdcca08f8c13d4354f0a7769d1e6119ff8bc5d77341354fd299bc28R97).

Reply:
jasně, to je tahle kulišárna, 🤦